<html>
<head><title>Matches for Mike Bada</title></head>
<body>
<a href="Class and span matcher.html">Class and span matcher</a>
<h1>Matches for Mike Bada</h1>
Each concept that was considered a match is shown in the text that it was found in.  The matching annotations from the other concept sets are also shown.
<hr>
<h2>CHEBI:14321</h2>
<h2>CHEBI:15428</h2>
<hr><p>Text source docID = 11604102<p>. The ability of Nova-1 to activate exon selection in neurons is antagonized by a second RNA-binding protein, brPTB (brain-enriched polypyrimidine tra<b>ct-bind</b>ing protein), which interacts with Nova-1 and inhibits its function [24]. DAZAP1 could function in a similar manner by binding to DAZL and inhibiting 
<ul><li>
ct-bind  <ul><li>Mike Bada</li><li>class = CHEBI:15428</li><li>spans = 13102|13109</li></ul><li>
ct-bind  <ul><li>Kristin Garcia</li><li>class = CHEBI:15428</li><li>spans = 13102|13109</li></ul></ul>
<h2>CHEBI:16449</h2>
<h2>CHEBI:16991</h2>
<hr><p>Text source docID = 11597317<p>by the break invade(s) the homologous sequence in the second mutant GFP copy, resulting in repair of the DSB via a homology-mediated gene conversion e<b>ven</b>t. The configuration of the GFP construct is such that homology-mediated repair often results in the formation of a functional copy of GFP. Such event
<ul><li>
ven  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 3918|3921</li></ul><li>
ven  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 3918|3921</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ions during the damage response. Further experiments are clearly required to clarify the functional interactions between BRCA2 and RAD51 during recomb<b>ina</b>tional repair. It should be relatively straightforward to determine whether the role of BRCA2 is limited to transporting RAD51 to the nucleus. If this
<ul><li>
ina  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 12046|12049</li></ul><li>
ina  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 12046|12049</li></ul></ul>
<hr><p>Text source docID = 11597317<p>1]. Finally, genetic measurements of recombination frequency have shown that Brca1-/- embryonic stem (ES) cells are deficient in recombinational DSB r<b>epa</b>ir [22,23]. The similarity between phenotypes associated with BRCA1 and BRCA2 deficiency, together with data showing a similar effect of DNA damage on
<ul><li>
epa  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 2773|2776</li></ul><li>
epa  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 2773|2776</li></ul></ul>
<hr><p>Text source docID = 11597317<p>d breaks. The second paper, by Davies et al, begins to address the mechanism through which BRCA2 makes its contribution to recombinational repair. BRC<b>A2 </b>appears to function in recombination via interactions with the major eukaryotic recombinase RAD51 [1,2,3]. We briefly review the context in which the 
<ul><li>
A2   <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 455|458</li></ul><li>
A2   <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 455|458</li></ul></ul>
<hr><p>Text source docID = 11604102<p>on proteins between thioredoxin and DAZAP1 were prepared and purified on His-Bind metal chelation resins (Novagen, Madison, WI). The proteins were mix<b>ed </b>with Freund's adjuvant and injected into rabbits to generate the anti-DAZAPl-C antibody. An oligopeptide containing the last 19 ammo acid residues of 
<ul><li>
ed   <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 18230|18233</li></ul><li>
ed   <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 18230|18233</li></ul></ul>
<hr><p>Text source docID = 11597317<p>en et al [27]. These investigators showed that expression of constructs containing the BRC4 repeat in MCF-7 cells enhances the radiosensitivity of cel<b>ls </b>and blocks both the G2/M delay associated with damage and the ability of the transfected cells to assembly subnuclear RAD51 foci. In the paper by Davi
<ul><li>
ls   <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 7585|7588</li></ul><li>
ls   <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 7585|7588</li></ul></ul>
<hr><p>Text source docID = 11604102<p> radiation hybrids containing mouse chromosome fragments in a hamster background were purchased from Research Genetics (Huntsville, AL). The presence <b>of </b>mouse Dazap1 in the radiation hybrids was determined by PCR and the results were sent to the MIT server  for computerized physical mapping of the gene
<ul><li>
of   <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 16708|16711</li></ul><li>
of   <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 16708|16711</li></ul></ul>
<hr><p>Text source docID = 11597317<p>age (reviewed in [30]). RAD51 paralogs in particular, including XRCC2, XRCC3, RAD51B, RAD51C and RAD51D, may function as a complex that actively promo<b>tes</b> RAD51 assembly [31]. BRCA2 could cooperate with RAD51 paralogs in promoting RAD51 assembly, or even provide an alternative assembly pathway. The obse
<ul><li>
tes  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 11281|11284</li></ul><li>
tes  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 11281|11284</li></ul></ul>
<hr><p>Text source docID = 11597317<p>nts [9,10,11]. These foci are probably sites of repair of spontaneous and induced DNA damage [12,13,14]. Cell lines defective in either BRCA1 or BRCA2<b> ar</b>e sensitive to damaging agents that form double-strand breaks (DSBs), as are other cell lines defective in recombinational repair (reviewed in [15]). 
<ul><li>
 ar  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 1955|1958</li></ul><li>
 ar  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 1955|1958</li></ul></ul>
<hr><p>Text source docID = 11597317<p> BRCA2 may thus promote RAD51 assembly into recombinational repair complexes via a negative regulatory mechanism (i.mouseEvent. by blocking RAD51–BRCA<b>2 i</b>nteraction until damage has occurred and factors required for 'productive' assembly of RAD51 at damaged sites are in place). As mentioned previously, 
<ul><li>
2 i  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 10554|10557</li></ul><li>
2 i  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 10554|10557</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ide was also reported to inhibit the RAD51–DNA interaction. These experiments raise the possibility that the full-length BRCA2 protein could act as a <b>neg</b>ative regulator of inappropriate RAD51–DNA interaction.

The final experiment reported by Davies et al [26] is particularly important with respect to 
<ul><li>
neg  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 8212|8215</li></ul><li>
neg  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 8212|8215</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ahan et al provide genetic data indicating a requirement for BRCA2 in homology-dependent (recombinational) repair of DNA double-strand breaks. The sec<b>ond</b> paper, by Davies et al, begins to address the mechanism through which BRCA2 makes its contribution to recombinational repair. BRCA2 appears to functi
<ul><li>
ond  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 322|325</li></ul><li>
ond  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 322|325</li></ul></ul>
<hr><p>Text source docID = 11597317<p>3,14]. Cell lines defective in either BRCA1 or BRCA2 are sensitive to damaging agents that form double-strand breaks (DSBs), as are other cell lines d<b>efe</b>ctive in recombinational repair (reviewed in [15]). BRCA2 interacts with the RAD51 recombinase via direct protein-protein contacts [16,17,18,19]. Bioc
<ul><li>
efe  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 2053|2056</li></ul><li>
efe  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 2053|2056</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ys and electron microscopy methods were used to show that a BRC3 peptide interferes with the ability of RAD51 to assemble into oligomeric filaments on<b> DN</b>A. A BRC4 peptide was also reported to inhibit the RAD51–DNA interaction. These experiments raise the possibility that the full-length BRCA2 protein c
<ul><li>
 DN  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 8045|8048</li></ul><li>
 DN  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 8045|8048</li></ul></ul>
<hr><p>Text source docID = 11604102<p>DAZAP1 protein, two antibodies against mouse DAZAP1 were generated. The anti DAZAP1-C antibody was raised against a recombinant protein containing the<b> C-</b>terminal proline-rich portion, and the anti DAZAP1-P antibody was raised against an oligopeptide containing the last 19 amino acid residue at the C-te
<ul><li>
 C-  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 9525|9528</li></ul><li>
 C-  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 9525|9528</li></ul></ul>
<hr><p>Text source docID = 11597317<p> These experiments raise the possibility that the full-length BRCA2 protein could act as a negative regulator of inappropriate RAD51–DNA interaction.
<b>
Th</b>e final experiment reported by Davies et al [26] is particularly important with respect to understanding the mechanism(s) through which BRCA2 promotes
<ul><li>

Th  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 8271|8274</li></ul><li>

Th  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 8271|8274</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ation hybrids containing mouse chromosome fragments in a hamster background were purchased from Research Genetics (Huntsville, AL). The presence of mo<b>use</b> Dazap1 in the radiation hybrids was determined by PCR and the results were sent to the MIT server  for computerized physical mapping of the gene.

Ex
<ul><li>
use  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 16713|16716</li></ul><li>
use  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 16713|16716</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ntains a pair of mutated GFP genes (GFP encodes the easily detected green fluorescent protein), one of which contains a restriction site for I-SceI, a<b> ye</b>ast intron encoded endonuclease with an 18 base pair recognition site. Transient transfection of an I-SceI expression vector results in the production
<ul><li>
 ye  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 3543|3546</li></ul><li>
 ye  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 3543|3546</li></ul></ul>
<hr><p>Text source docID = 11597317<p>

The final experiment reported by Davies et al [26] is particularly important with respect to understanding the mechanism(s) through which BRCA2 prom<b>ote</b>s RAD51-dependent recombination. The localization of RAD51 was examined in CAPAN-1 cells, the same human tumor line examined by Moynahan et al [24]. I
<ul><li>
ote  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 8420|8423</li></ul><li>
ote  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 8420|8423</li></ul></ul>
<hr><p>Text source docID = 11604102<p>s and produce much larger (if any) fragments from genomic DNA.

To study the expression of the DAZAP1 protein, two antibodies against mouse DAZAP1 wer<b>e g</b>enerated. The anti DAZAP1-C antibody was raised against a recombinant protein containing the C-terminal proline-rich portion, and the anti DAZAP1-P an
<ul><li>
e g  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 9430|9433</li></ul><li>
e g  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 9430|9433</li></ul></ul>
<hr><p>Text source docID = 11597317<p>stribution of BRCA1 and BRCA2 in repair-proficient cells, led to the hypothesis that BRCA2, like BRCA1, is required for efficient recombinational repa<b>ir.</b> The paper by Moynahan et al [24] provides important support for this hypothesis.

Measuring DSB-induced recombination frequency in BRCA2-defective ce
<ul><li>
ir.  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 3079|3082</li></ul><li>
ir.  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 3079|3082</li></ul></ul>
<hr><p>Text source docID = 11597317<p>1 and BRCA2 form discrete nuclear foci during the S phase and after exposure to DNA damaging agents [9,10,11]. These foci are probably sites of repair<b> of</b> spontaneous and induced DNA damage [12,13,14]. Cell lines defective in either BRCA1 or BRCA2 are sensitive to damaging agents that form double-strand
<ul><li>
 of  <ul><li>Mike Bada</li><li>class = CHEBI:16991</li><li>spans = 1859|1862</li></ul><li>
 of  <ul><li>Kristin Garcia</li><li>class = CHEBI:16991</li><li>spans = 1859|1862</li></ul></ul>
<h2>CHEBI:18367</h2>
<h2>CHEBI:23525</h2>
<h2>CHEBI:23526</h2>
<h2>CHEBI:23888</h2>
<h2>CHEBI:24459</h2>
<h2>CHEBI:24460</h2>
<h2>CHEBI:26130</h2>
<h2>CHEBI:26271</h2>
<h2>CHEBI:27243</h2>
<h2>CHEBI:27244</h2>
<h2>CHEBI:27570</h2>
<hr><p>Text source docID = 11604102<p>ovagen, Madison, WI). The proteins were mixed with Freund's adjuvant and injected into rabbits to generate the anti-DAZAPl-C antibody. An oligopeptide<b> co</b>ntaining the last 19 ammo acid residues of the mouse DAZAP1 was synthesized in vitro using the services of Bethyl Laboratories (Montgomery, TX). The p
<ul><li>
 co  <ul><li>Mike Bada</li><li>class = CHEBI:27570</li><li>spans = 18337|18340</li></ul><li>
 co  <ul><li>Kristin Garcia</li><li>class = CHEBI:27570</li><li>spans = 18337|18340</li></ul></ul>
<h2>CHEBI:29987</h2>
<h2>CHEBI:30360</h2>
<hr><p>Text source docID = 11319941<p>nly two alleles, A and B, or A and D. All stocks of mice were obtained from the Jackson Laboratory . ABF2 mice were generated at the University of Ten<b>nessee by D</b>r. Richelle Strom [56] using Jackson Laboratory foundation stock. The F2 mice ranged in age from 35 to 143 days. The standard inbred strains ranged in
<ul><li>
nessee by D  <ul><li>Mike Bada</li><li>class = CHEBI:30360</li><li>spans = 26307|26318</li></ul><li>
nessee by D  <ul><li>Kristin Garcia</li><li>class = CHEBI:30360</li><li>spans = 26307|26318</li></ul></ul>
<h2>CHEBI:30755</h2>
<h2>CHEBI:30756</h2>
<h2>CHEBI:30758</h2>
<h2>CHEBI:30759</h2>
<h2>CHEBI:32441</h2>
<h2>CHEBI:32484</h2>
<h2>CHEBI:32874</h2>
<h2>CHEBI:33697</h2>
<hr><p>Text source docID = 11604102<p> mental retardation protein [21]. Both FXR1P and FXR2P are associated with the polyribosomes [22].

The significance of the interaction between DAZAP1<b> an</b>d DAZL/DAZ remains to be defined. These proteins may act together to facilitate the expression of a set of genes in germ cells. For example, DAZAP1 co
<ul><li>
 an  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 12341|12344</li></ul><li>
 an  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 12341|12344</li></ul></ul>
<hr><p>Text source docID = 11604102<p>nd to participate in many cellular functions, including RNA transcription, pre-mRNA processing, mRNA transport, localization, translation and stabilit<b>y [</b>20]. A role for the DAZ family in the regulation of mRNA translation is supported by lines of circumstantial evidence, including the association of DA
<ul><li>
y [  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 11715|11718</li></ul><li>
y [  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 11715|11718</li></ul></ul>
<hr><p>Text source docID = 11604102<p> and results in a shorter Prrp with a different C-terminal end sequence.

Figure 1

Evolutionary conservation of the DAZAP1 proteins. The amino acid s<b>equ</b>ences of the human and mouse DAZAP1s and the Xenopus Prrp are compared. The two RNA binding domains are boxed. Differences between the human and the m
<ul><li>
equ  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 5910|5913</li></ul><li>
equ  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 5910|5913</li></ul></ul>
<hr><p>Text source docID = 11604102<p>R2P, that were identified through their interaction with another polysomal-associated RNA-binding protein, the fragile X mental retardation protein [2<b>1].</b> Both FXR1P and FXR2P are associated with the polyribosomes [22].

The significance of the interaction between DAZAP1 and DAZL/DAZ remains to be defin
<ul><li>
1].  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 12221|12224</li></ul><li>
1].  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 12221|12224</li></ul></ul>
<hr><p>Text source docID = 11604102<p>sing, mRNA transport, localization, translation and stability [20]. A role for the DAZ family in the regulation of mRNA translation is supported by li<b>nes</b> of circumstantial evidence, including the association of DAZL with polyribosomes [12]. The absence of DAZAP1 from polyribosomes indicates that it is 
<ul><li>
nes  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 11805|11808</li></ul><li>
nes  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 11805|11808</li></ul></ul>
<hr><p>Text source docID = 11604102<p>olated and characterized the mouse Dazap1 gene, and studied its expression and the subcellular localization of its protein product.

Results

The huma<b>n a</b>nd mouse genes have similar genomic structures and map to syntenic chromosomal regions. The mouse and human DAZAP1 proteins share 98% identity and the
<ul><li>
n a  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 844|847</li></ul><li>
n a  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 844|847</li></ul></ul>
<hr><p>Text source docID = 11604102<p>E is expressed later, in the cytoplasm of pachytene spermatocytes [5]. Genetic and biochemical studies suggest a role for the DAZ family in the regula<b>tio</b>n of mRNA translation. Drosophila Boule mutants was defective in the translation of the meiosis-specific CDC25 homologue, Twine [11], and DAZL was fou
<ul><li>
tio  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 3171|3174</li></ul><li>
tio  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 3171|3174</li></ul></ul>
<hr><p>Text source docID = 11604102<p>g, #734–718) and an adaptor primer as the primers for PCR amplification. The PCR products were cloned into a TA cloning vector pCR2.1-TOPO (Invitrogen<b>, C</b>arlsbad CA). Dazap1 clones were identified by colony hybridization and sequenced. The 5' RACE clone with the longest 5' UTR region and the cDNA clone 
<ul><li>
, C  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 15847|15850</li></ul><li>
, C  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 15847|15850</li></ul></ul>
<hr><p>Text source docID = 11604102<p>rotein product. The subcellular localization of DAZAP1 suggests that it is not involved directly in mRNA translation.

Results

Characterization of th<b>e m</b>ouse Dazap1 cDNA

Mouse Dazap1 cDNA clones were isolated by library screening, and the 5' end of the cDNA was isolated by 5' RACE [15]. The near fall 
<ul><li>
e m  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 4297|4300</li></ul><li>
e m  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 4297|4300</li></ul></ul>
<hr><p>Text source docID = 11604102<p>Several overlapping lambda clones containing mouse Dazap1 genomic sequences were isolated. The locations of exons were determined by PCR amplification<b> ac</b>ross exon-intron boundaries following by sequencing. All but the first exon were isolated and mapped. The genomic structure of the human DAZAP1 gene w
<ul><li>
 ac  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 6368|6371</li></ul><li>
 ac  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 6368|6371</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ale infertility factor, DAZ (Deleted in Azoospermia). In vitro, DAZAP1 interacts with both the Y chromosome-encoded DAZ and an autosome-encoded DAZ-li<b>ke </b>protein, DAZL. DAZAP1 contains two RNA-binding domains (RBDs) and a proline-rich C-terminal portion, and is expressed most abundantly in the testis. T
<ul><li>
ke   <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 430|433</li></ul><li>
ke   <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 430|433</li></ul></ul>
<hr><p>Text source docID = 11604102<p>day 17 (El 7) embryos, new born mice (Day 0), and mice at various days after birth. Wv/Wv testes contain diminished germ cell population due to a muta<b>ted </b>W (White spotted) gene. GC1 and MT4 are mouse germ cell and Sertoli cell lines, respectively, and gDNA is mouse genomic DNA. The PCR primers span over
<ul><li>
ted   <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 9119|9123</li></ul><li>
ted   <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 9119|9123</li></ul></ul>
<hr><p>Text source docID = 11604102<p>preference for poly U and poly G. Its natural substrates have not been identified. Recently, the Xenopus orthologue of DAZAP1, Prrp, was identified an<b>d c</b>haracterized [16]. Prrp binds to a 340 nt sequence in the 3' UTR of Xenopus Vg1 mRNA. This Vg1 localization element (VLE) is sufficient for the migrat
<ul><li>
d c  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 13965|13968</li></ul><li>
d c  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 13965|13968</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ns/insertions. The mammalian proteins shares 89% similarity and 81% identity with Xenopus Prrp (for proline-rich RNA binding protein) [16]. The two RB<b>Ds </b>are especially highly conserved. They share 98% and 97% similarity and 97% and 92% identity, respectively, between DAZAP1 and Prrp. These proteins may
<ul><li>
Ds   <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 5355|5358</li></ul><li>
Ds   <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 5355|5358</li></ul></ul>
<hr><p>Text source docID = 11604102<p>orming growth factor-β family that is required for generating dorsal mesoderm at the blastula stage of Xenopus embryogenesis [25]. Sequence conservati<b>on </b>between the RBDs of DAZAP1 and Prrp suggests that these proteins may bind to similar RNA sequences. However, a BLAST search of the GenBank for the 340
<ul><li>
on   <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 14565|14568</li></ul><li>
on   <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 14565|14568</li></ul></ul>
<hr><p>Text source docID = 11604102<p>arch of the GenBank for the 340 nt VLE sequence failed to identify any mammalian sequences with significant homology. Further mapping of the RNA seque<b>nce</b> within VLE that binds Prrp, and possibly DAZAP1, may help to identify the natural substrates of DAZAP1.

Conclusions

DAZAP1 is an evolutionarily con
<ul><li>
nce  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 14837|14840</li></ul><li>
nce  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 14837|14840</li></ul></ul>
<hr><p>Text source docID = 11604102<p> Prrp, and possibly DAZAP1, may help to identify the natural substrates of DAZAP1.

Conclusions

DAZAP1 is an evolutionarily conserved RNA-binding pro<b>tei</b>n. It is present at variable levels in many tissues. Its predominant expression in testes suggests a role in spermatogenesis. In mouse testes, DAZAP1 
<ul><li>
tei  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 15012|15015</li></ul><li>
tei  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 15012|15015</li></ul></ul>
<hr><p>Text source docID = 11604102<p>enriched polypyrimidine tract-binding protein), which interacts with Nova-1 and inhibits its function [24]. DAZAP1 could function in a similar manner <b>by </b>binding to DAZL and inhibiting its function. Comparing the phenotypes of Dazl1 and Dazap1 single and double knock-out mice may provide some clues to t
<ul><li>
by   <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 13225|13228</li></ul><li>
by   <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 13225|13228</li></ul></ul>
<hr><p>Text source docID = 11604102<p> at variable levels in many tissues. Its predominant expression in testes suggests a role in spermatogenesis. In mouse testes, DAZAP1 was found both i<b>n t</b>he nuclei and in the cytoplasm. Its absence from polyribosomes indicates that it is not directly involved in mRNA translation.

Materials and methods

<ul><li>
n t  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 15181|15184</li></ul><li>
n t  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 15181|15184</li></ul></ul>
<hr><p>Text source docID = 11604102<p>g specificity. The C-terminal proline-rich portions of DAZAP1 and Prrp are less conserved (81% similarity and 71% identity). There is an insertion of <b>a 5</b>8 bp segment in Prrp cDNA that causes a change of reading frame and results in a shorter Prrp with a different C-terminal end sequence.

Figure 1

Evo
<ul><li>
a 5  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 5694|5697</li></ul><li>
a 5  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 5694|5697</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ation of the mouse Dazap1 cDNA

Mouse Dazap1 cDNA clones were isolated by library screening, and the 5' end of the cDNA was isolated by 5' RACE [15]. <b>The</b> near fall length cDNA consists of a 53 bp 5' untranslated region (UTR), an open reading frame for a protein of 405 amino acid residues, and a 362 bp 
<ul><li>
The  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 4436|4439</li></ul><li>
The  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 4436|4439</li></ul></ul>
<hr><p>Text source docID = 11604102<p>d by the neuron-specific nuclear RNA-binding protein, Nova-1. Nova-1 regulates the alternative splicing of the pre-mRNAs encoding neuronal inhibitory <b>gly</b>cine receptor α2 (GlyR α2) [23]. The ability of Nova-1 to activate exon selection in neurons is antagonized by a second RNA-binding protein, brPTB (br
<ul><li>
gly  <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 12918|12921</li></ul><li>
gly  <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 12918|12921</li></ul></ul>
<hr><p>Text source docID = 11604102<p>1 regulates the alternative splicing of the pre-mRNAs encoding neuronal inhibitory glycine receptor α2 (GlyR α2) [23]. The ability of Nova-1 to activa<b>te </b>exon selection in neurons is antagonized by a second RNA-binding protein, brPTB (brain-enriched polypyrimidine tract-binding protein), which interacts
<ul><li>
te   <ul><li>Mike Bada</li><li>class = CHEBI:33697</li><li>spans = 12985|12988</li></ul><li>
te   <ul><li>Kristin Garcia</li><li>class = CHEBI:33697</li><li>spans = 12985|12988</li></ul></ul>
<h2>CHEBI:33699</h2>
<hr><p>Text source docID = 11604102<p>t (VLE) is sufficient for the migration and clustering of Vg1 mRNA to the vegetal cortex of mature oocyte. Prrp also interacts through its proline-ric<b>h do</b>main with two microfilament-associated proteins profilin and Mena, which may facilitate the anchoring of Vg1 mRNA to the vegetal cortex. The Vg1 RNA e
<ul><li>
h do  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 14232|14236</li></ul><li>
h do  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 14232|14236</li></ul></ul>
<hr><p>Text source docID = 11604102<p> Dazl1 and Dazap1 persisted throughout testes development, in both the prenatal and postnatal periods. Dazl1 and Dazap1 transcripts were also present <b>in t</b>he testes of Wv/Wv mutant mice which contained diminished number of germ cells [18]. However, only Dazap1 was expressed in a mouse germ cell line GCl-
<ul><li>
in t  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 8301|8305</li></ul><li>
in t  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 8301|8305</li></ul></ul>
<hr><p>Text source docID = 11604102<p> glycine receptor α2 (GlyR α2) [23]. The ability of Nova-1 to activate exon selection in neurons is antagonized by a second RNA-binding protein, brPTB<b> (bra</b>in-enriched polypyrimidine tract-binding protein), which interacts with Nova-1 and inhibits its function [24]. DAZAP1 could function in a similar mann
<ul><li>
 (bra  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 13067|13072</li></ul><li>
 (bra  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 13067|13072</li></ul></ul>
<hr><p>Text source docID = 11604102<p> a 53 bp 5' untranslated region (UTR), an open reading frame for a protein of 405 amino acid residues, and a 362 bp 3' UTR (GenBank Accession No: AF22<b>5910</b>). The coding region shares 89% similarity with that of the human orthologue. The 3' UTR sequence is remarkably conserved. It contains three segments 
<ul><li>
5910  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 4623|4627</li></ul><li>
5910  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 4623|4627</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ethods

Isolation of mouse Dazap1 cDNA clones

Dazap1 cDNA clones were isolated from a mouse testis cDNA library (#937308, Stratagene, La Jolla, CA) u<b>sing</b> a human DAZAP1 cDNA as a probe. The 5' end of the cDNA was isolated by 5' RACE [15], using prdap11 (ttgcgggccatatccttg, #749–732) as the primer for c
<ul><li>
sing  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 15477|15481</li></ul><li>
sing  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 15477|15481</li></ul></ul>
<hr><p>Text source docID = 11604102<p>s) [14]. One of them, DAZAP1, is expressed predominantly in testes. It encodes a protein with two RNA binding domains and a proline rich C-terminal po<b>rtio</b>n. The DAZAP1 protein interacted with both DAZ and DAZL in vitro. It also bound to RNA homopolymers. We now report our characterization of the mouse D
<ul><li>
rtio  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 3973|3977</li></ul><li>
rtio  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 3973|3977</li></ul></ul>
<hr><p>Text source docID = 11604102<p>was shown both in vitro and in a yeast three-hybrid system to bind specifically to oligo(U) stretches interspersed by G or C residues, including a U-r<b>ich </b>segment in the 5' UTR of mouse Cdc25C mRNA [13].

In an attempt to elucidate the function of the DAZ gene family and to understanding the mechanisms o
<ul><li>
ich   <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 3555|3559</li></ul><li>
ich   <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 3555|3559</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ocalization, translation and stability [20]. A role for the DAZ family in the regulation of mRNA translation is supported by lines of circumstantial e<b>vide</b>nce, including the association of DAZL with polyribosomes [12]. The absence of DAZAP1 from polyribosomes indicates that it is not directly involved in
<ul><li>
vide  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 11828|11832</li></ul><li>
vide  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 11828|11832</li></ul></ul>
<hr><p>Text source docID = 11604102<p>re oocyte. Prrp also interacts through its proline-rich domain with two microfilament-associated proteins profilin and Mena, which may facilitate the <b>anch</b>oring of Vg1 mRNA to the vegetal cortex. The Vg1 RNA encodes a member of the transforming growth factor-β family that is required for generating dorsa
<ul><li>
anch  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 14328|14332</li></ul><li>
anch  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 14328|14332</li></ul></ul>
<hr><p>Text source docID = 11604102<p>s at embryonic day 15, similar to Dazl1 mRNA (Figure 3). The expression of both Dazl1 and Dazap1 persisted throughout testes development, in both the <b>pren</b>atal and postnatal periods. Dazl1 and Dazap1 transcripts were also present in the testes of Wv/Wv mutant mice which contained diminished number of ger
<ul><li>
pren  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 8222|8226</li></ul><li>
pren  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 8222|8226</li></ul></ul>
<hr><p>Text source docID = 11604102<p> Alternatively, DAZL and DAZAP1 may act antagonistically to regulate the timing and the level of expression. Such an antagonistic interaction between <b>two i</b>nteracting RNA-binding proteins is exemplified by the neuron-specific nuclear RNA-binding protein, Nova-1. Nova-1 regulates the alternative splicing o
<ul><li>
two i  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 12718|12723</li></ul><li>
two i  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 12718|12723</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ines of circumstantial evidence, including the association of DAZL with polyribosomes [12]. The absence of DAZAP1 from polyribosomes indicates that it<b> is </b>not directly involved in protein synthesis. This finding is different from two RNA-binding proteins, FXR1P and FXR2P, that were identified through the
<ul><li>
 is   <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 11954|11958</li></ul><li>
 is   <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 11954|11958</li></ul></ul>
<hr><p>Text source docID = 11604102<p>slation and stability [20]. A role for the DAZ family in the regulation of mRNA translation is supported by lines of circumstantial evidence, includin<b>g th</b>e association of DAZL with polyribosomes [12]. The absence of DAZAP1 from polyribosomes indicates that it is not directly involved in protein synthesi
<ul><li>
g th  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 11845|11849</li></ul><li>
g th  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 11845|11849</li></ul></ul>
<hr><p>Text source docID = 11604102<p>1 RNA encodes a member of the transforming growth factor-β family that is required for generating dorsal mesoderm at the blastula stage of Xenopus emb<b>ryog</b>enesis [25]. Sequence conservation between the RBDs of DAZAP1 and Prrp suggests that these proteins may bind to similar RNA sequences. However, a BLAS
<ul><li>
ryog  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 14529|14533</li></ul><li>
ryog  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 14529|14533</li></ul></ul>
<hr><p>Text source docID = 11604102<p>process is vital for the propagation of a species, and involves a large portion of the genome of an organism to ensure the quality and quantity of the<b> fin</b>al products. It is estimated that mutations in up to 11% of all genes in Drosophila might lead to male sterility [1]. This is likely to be true for hu
<ul><li>
 fin  <ul><li>Mike Bada</li><li>class = CHEBI:33699</li><li>spans = 1919|1923</li></ul><li>
 fin  <ul><li>Kristin Garcia</li><li>class = CHEBI:33699</li><li>spans = 1919|1923</li></ul></ul>
<h2>CHEBI:33791</h2>
<h2>CHEBI:35222</h2>
<h2>CHEBI:37527</h2>
<hr><p>Text source docID = 11532192<p>8], except that 5 days of training were used. The mice were analyzed in the same procedure room as was used for IOP measurment.

Clinical examinations<b>

Fo</b>r most strains, anterior chambers were examined with a slit lamp biomicroscope [16]. At least 8 mice of each strain shown in figure 1 were evaluated. 
<ul><li>


Fo  <ul><li>Mike Bada</li><li>class = CHEBI:37527</li><li>spans = 37924|37928</li></ul><li>


Fo  <ul><li>Kristin Garcia</li><li>class = CHEBI:37527</li><li>spans = 37924|37928</li></ul></ul>
<h2>CHEBI:37670</h2>
<hr><p>Text source docID = 11604102<p>e Protease Inhibitors Set (Roche Molecular Biochemicals, Indianapolis, IN). Homogenates were centrifuged at 1,000 × g for 10 minutes to pellet cell de<b>bris and nuclei. Af</b>ter an additional centrifugation at 10,000 × g for 10 minutes to pellet the mitochondria, aliquots of the supernatant were applied to 15–45% sucrose g
<ul><li>
bris and nuclei. Af  <ul><li>Mike Bada</li><li>class = CHEBI:37670</li><li>spans = 19941|19960</li></ul><li>
bris and nuclei. Af  <ul><li>Kristin Garcia</li><li>class = CHEBI:37670</li><li>spans = 19941|19960</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ecular Biochemicals, Indianapolis, IN). Homogenates were centrifuged at 1,000 × g for 10 minutes to pellet cell debris and nuclei. After an additional<b> centrifugation at </b>10,000 × g for 10 minutes to pellet the mitochondria, aliquots of the supernatant were applied to 15–45% sucrose gradients in 20 mM Tris, 100 mM KCl a
<ul><li>
 centrifugation at   <ul><li>Mike Bada</li><li>class = CHEBI:37670</li><li>spans = 19977|19996</li></ul><li>
 centrifugation at   <ul><li>Kristin Garcia</li><li>class = CHEBI:37670</li><li>spans = 19977|19996</li></ul></ul>
<h2>CHEBI:38867</h2>
<hr><p>Text source docID = 11532192<p>however, males infrequently had significantly higher IOP than females. For example, in one experiment, B6 males had an average IOP of 14.2 ± 0.3 mmHg <b>(n = 12) w</b>hereas the average IOP of females was 13.1 ± 0.3 mmHg (n = 12, P = 0.01). If real, this sporadic sex difference was not dependent on age, sometimes oc
<ul><li>
(n = 12) w  <ul><li>Mike Bada</li><li>class = CHEBI:38867</li><li>spans = 12466|12476</li></ul><li>
(n = 12) w  <ul><li>Kristin Garcia</li><li>class = CHEBI:38867</li><li>spans = 12466|12476</li></ul></ul>
<h2>CHEBI:46883</h2>
<h2>CHEBI:4705</h2>
<hr><p>Text source docID = 11597317<p>ns to prevent sequestration of RAD51 in a nonfunctional form. Alternatively, or in addition, suppression of RAD51–ssDNA interaction might be important<b> for </b>preventing RAD51 from binding to the regions of ssDNA that form during normal DNA replication. BRCA2 may thus promote RAD51 assembly into recombinatio
<ul><li>
 for   <ul><li>Mike Bada</li><li>class = CHEBI:4705</li><li>spans = 10305|10310</li></ul><li>
 for   <ul><li>Kristin Garcia</li><li>class = CHEBI:4705</li><li>spans = 10305|10310</li></ul></ul>
<hr><p>Text source docID = 11597317<p>his result could mean that regulatory factors, such as BRCA2, are required to suppress RAD51–dsDNA interactions to prevent sequestration of RAD51 in a<b> nonf</b>unctional form. Alternatively, or in addition, suppression of RAD51–ssDNA interaction might be important for preventing RAD51 from binding to the regi
<ul><li>
 nonf  <ul><li>Mike Bada</li><li>class = CHEBI:4705</li><li>spans = 10196|10201</li></ul><li>
 nonf  <ul><li>Kristin Garcia</li><li>class = CHEBI:4705</li><li>spans = 10196|10201</li></ul></ul>
<hr><p>Text source docID = 11597317<p> double-strand DNA; ES = embryonic stem; GFP = green fluorescent protein; ssDNA = single-strand DNA.

Figures and Tables

Figure 1

Recombination subs<b>trate</b>s used for assaying homology-directed repair. Cutting at the I-SceI site within the mutant GFP (SceGFP) results in a double-strand break that can be r
<ul><li>
trate  <ul><li>Mike Bada</li><li>class = CHEBI:4705</li><li>spans = 12909|12914</li></ul><li>
trate  <ul><li>Kristin Garcia</li><li>class = CHEBI:4705</li><li>spans = 12909|12914</li></ul></ul>
<hr><p>Text source docID = 11597317<p>A (dsDNA) [3,29]. This result could mean that regulatory factors, such as BRCA2, are required to suppress RAD51–dsDNA interactions to prevent sequestr<b>ation of RAD51 in</b> a nonfunctional form. Alternatively, or in addition, suppression of RAD51–ssDNA interaction might be important for preventing RAD51 from binding to t
<ul><li>
ation of RAD51 in  <ul><li>Mike Bada</li><li>class = CHEBI:4705</li><li>spans = 10177|10194</li></ul><li>
ation of RAD51 in  <ul><li>Kristin Garcia</li><li>class = CHEBI:4705</li><li>spans = 10177|10194</li></ul></ul>
<hr><p>Text source docID = 11597317<p>strand DNA; ES = embryonic stem; GFP = green fluorescent protein; ssDNA = single-strand DNA.

Figures and Tables

Figure 1

Recombination substrates u<b>sed for assaying </b>homology-directed repair. Cutting at the I-SceI site within the mutant GFP (SceGFP) results in a double-strand break that can be repaired through homo
<ul><li>
sed for assaying   <ul><li>Mike Bada</li><li>class = CHEBI:4705</li><li>spans = 12917|12934</li></ul><li>
sed for assaying   <ul><li>Kristin Garcia</li><li>class = CHEBI:4705</li><li>spans = 12917|12934</li></ul></ul>
<h2>CHEBI:9160</h2>
<hr><p>Text source docID = 11597317<p>n might be important for preventing RAD51 from binding to the regions of ssDNA that form during normal DNA replication. BRCA2 may thus promote RAD51 a<b>ssemb</b>ly into recombinational repair complexes via a negative regulatory mechanism (i.mouseEvent. by blocking RAD51–BRCA2 interaction until damage has occur
<ul><li>
ssemb  <ul><li>Mike Bada</li><li>class = CHEBI:9160</li><li>spans = 10435|10440</li></ul><li>
ssemb  <ul><li>Kristin Garcia</li><li>class = CHEBI:9160</li><li>spans = 10435|10440</li></ul></ul>
<hr><p>Text source docID = 11597317<p> single-strand DNA.

Figures and Tables

Figure 1

Recombination substrates used for assaying homology-directed repair. Cutting at the I-SceI site wit<b>hin t</b>he mutant GFP (SceGFP) results in a double-strand break that can be repaired through homologous gene conversion using a 3'-truncated copy of GFP as se
<ul><li>
hin t  <ul><li>Mike Bada</li><li>class = CHEBI:9160</li><li>spans = 12990|12995</li></ul><li>
hin t  <ul><li>Kristin Garcia</li><li>class = CHEBI:9160</li><li>spans = 12990|12995</li></ul></ul>
<hr><p>Text source docID = 11597317<p>le-strand DNA (dsDNA) [3,29]. This result could mean that regulatory factors, such as BRCA2, are required to suppress RAD51–dsDNA interactions to prev<b>ent s</b>equestration of RAD51 in a nonfunctional form. Alternatively, or in addition, suppression of RAD51–ssDNA interaction might be important for preventing
<ul><li>
ent s  <ul><li>Mike Bada</li><li>class = CHEBI:9160</li><li>spans = 10165|10170</li></ul><li>
ent s  <ul><li>Kristin Garcia</li><li>class = CHEBI:9160</li><li>spans = 10165|10170</li></ul></ul>
<hr><p>Text source docID = 11597317<p>during normal DNA replication. BRCA2 may thus promote RAD51 assembly into recombinational repair complexes via a negative regulatory mechanism (i.mous<b>eEven</b>t. by blocking RAD51–BRCA2 interaction until damage has occurred and factors required for 'productive' assembly of RAD51 at damaged sites are in place
<ul><li>
eEven  <ul><li>Mike Bada</li><li>class = CHEBI:9160</li><li>spans = 10524|10529</li></ul><li>
eEven  <ul><li>Kristin Garcia</li><li>class = CHEBI:9160</li><li>spans = 10524|10529</li></ul></ul>
<hr><p>Text source docID = 11597317<p>A (ssDNA) over double-strand DNA (dsDNA) [3,29]. This result could mean that regulatory factors, such as BRCA2, are required to suppress RAD51–dsDNA i<b>nteractions to pr</b>event sequestration of RAD51 in a nonfunctional form. Alternatively, or in addition, suppression of RAD51–ssDNA interaction might be important for pre
<ul><li>
nteractions to pr  <ul><li>Mike Bada</li><li>class = CHEBI:9160</li><li>spans = 10146|10163</li></ul><li>
nteractions to pr  <ul><li>Kristin Garcia</li><li>class = CHEBI:9160</li><li>spans = 10146|10163</li></ul></ul>
<hr><p>Text source docID = 11597317<p>strand DNA.

Figures and Tables

Figure 1

Recombination substrates used for assaying homology-directed repair. Cutting at the I-SceI site within the <b>mutant GFP (SceGF</b>P) results in a double-strand break that can be repaired through homologous gene conversion using a 3'-truncated copy of GFP as sequence donor. The me
<ul><li>
mutant GFP (SceGF  <ul><li>Mike Bada</li><li>class = CHEBI:9160</li><li>spans = 12998|13015</li></ul><li>
mutant GFP (SceGF  <ul><li>Kristin Garcia</li><li>class = CHEBI:9160</li><li>spans = 12998|13015</li></ul></ul>
<h2>CHEBI_15377</h2>
<hr><p>Text source docID = 11532192<p>large scale genetic studies of factors determining IOP. Mice have great potential for helping to characterize the molecular mechanisms affecting IOP.
<b>
Mate</b>rials and Methods

Animal husbandry

All experiments were performed in compliance with the ARVO statement for use of animals in ophthalmic and vision 
<ul><li>

Mate  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_15377</li><li>spans = 34648|34653</li></ul><li>

Mate  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_15377</li><li>spans = 34648|34653</li></ul></ul>
<hr><p>Text source docID = 11319941<p>B, or A and D. All stocks of mice were obtained from the Jackson Laboratory . ABF2 mice were generated at the University of Tennessee by Dr. Richelle <b>Strom</b> [56] using Jackson Laboratory foundation stock. The F2 mice ranged in age from 35 to 143 days. The standard inbred strains ranged in age from 51 to 3
<ul><li>
Strom  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_15377</li><li>spans = 26330|26335</li></ul><li>
Strom  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_15377</li><li>spans = 26330|26335</li></ul></ul>
<h2>CHEBI_16670</h2>
<hr><p>Text source docID = 11597317<p> reported to inhibit the RAD51–DNA interaction. These experiments raise the possibility that the full-length BRCA2 protein could act as a negative reg<b>ulator </b>of inappropriate RAD51–DNA interaction.

The final experiment reported by Davies et al [26] is particularly important with respect to understanding th
<ul><li>
ulator   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 8224|8231</li></ul><li>
ulator   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 8224|8231</li></ul></ul>
<hr><p>Text source docID = 11597317<p> to repress the RAD51–BRCA2 interaction could reflect a role of full-length protein in suppressing unwanted RAD51–BRCA2 interactions.

Yet another pot<b>ential </b>role for BRCA2 in promoting RAD51-dependent recombinational repair is a positive role in assembly of RAD51 at damaged sites (Fig. 2b). Several protein
<ul><li>
ential   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 10885|10892</li></ul><li>
ential   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 10885|10892</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ng of one of the eight BRC repeats from human BRCA2. Expression of a single BRC repeat (BRC4) had previously been reported to act as an inhibitor of D<b>NA repa</b>ir by Chen et al [27]. These investigators showed that expression of constructs containing the BRC4 repeat in MCF-7 cells enhances the radiosensitivit
<ul><li>
NA repa  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 7420|7427</li></ul><li>
NA repa  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 7420|7427</li></ul></ul>
<hr><p>Text source docID = 11604102<p>arrier and injected into a goat. The anti-DAZAP1-P antibody thus produced was purified on an affinity column containing the oligopeptide antigen.

Wes<b>tern bl</b>otting

Mouse tissues were homogenized in the RIPA lysis buffer (150 mM NaCl, 1.0% NP-40, 0.5% DOC, 0.1% SDS, 50 mM Tris, pH 8.0) at a concentration o
<ul><li>
tern bl  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 18673|18680</li></ul><li>
tern bl  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 18673|18680</li></ul></ul>
<hr><p>Text source docID = 11597317<p>es with the ability of RAD51 to assemble into oligomeric filaments on DNA. A BRC4 peptide was also reported to inhibit the RAD51–DNA interaction. Thes<b>e exper</b>iments raise the possibility that the full-length BRCA2 protein could act as a negative regulator of inappropriate RAD51–DNA interaction.

The final e
<ul><li>
e exper  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 8126|8133</li></ul><li>
e exper  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_16670</li><li>spans = 8126|8133</li></ul></ul>
<h2>CHEBI_17544</h2>
<hr><p>Text source docID = 11532192<p>ds and cones [70]. Interestingly, the IOP of strain CBA/CaJ, which has one of the highest daytime IOPs does not appear to increase in the dark. Furthe<b>r more deta</b>iled studies are needed to define the characteristics of the diurnal rhythm of intraocular pressure in mice, and to determine whether it is lacking or
<ul><li>
r more deta  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17544</li><li>spans = 31251|31262</li></ul><li>
r more deta  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17544</li><li>spans = 31251|31262</li></ul></ul>
<h2>CHEBI_17992</h2>
<hr><p>Text source docID = 11604102<p>at DAZAP1 is not associated with polyribosomes. The post-mitochondrial supernatant of mouse testis extracts was analyzed on a 15–45% sucrose gradient.<b> Sedime</b>ntation was from left to right. The presence of DAZAP1 and DAZL in each fractions was analyzed by Western blotting.

Discussion

RNA-binding proteins 
<ul><li>
 Sedime  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17992</li><li>spans = 11395|11402</li></ul><li>
 Sedime  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17992</li><li>spans = 11395|11402</li></ul></ul>
<hr><p>Text source docID = 11604102<p>00 mM KCl and 5 mM MgCl2 and centrifuged in a Beckman SW41 rotor at 39,000 rpm for 2 hours at 4°C. Fractions of 0.5 ml were collected from the bottom <b>of the </b>tubes and analyzed by western blotting.

Acknowledgments

We thank Gary Kuo for his involvement in the construction of Dazap1 expression vectors, and 
<ul><li>
of the   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17992</li><li>spans = 20285|20292</li></ul><li>
of the   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17992</li><li>spans = 20285|20292</li></ul></ul>
<hr><p>Text source docID = 11604102<p>st- mitochondria fraction showed that, unlike DAZL, DAZAP1 did not co-sediment with polyribosomes (Figure 6).

Figure 6

Sucrose gradient analyses sho<b>ws that</b> DAZAP1 is not associated with polyribosomes. The post-mitochondrial supernatant of mouse testis extracts was analyzed on a 15–45% sucrose gradient. S
<ul><li>
ws that  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17992</li><li>spans = 11240|11247</li></ul><li>
ws that  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17992</li><li>spans = 11240|11247</li></ul></ul>
<hr><p>Text source docID = 11604102<p>s from left to right. The presence of DAZAP1 and DAZL in each fractions was analyzed by Western blotting.

Discussion

RNA-binding proteins have been <b>found t</b>o participate in many cellular functions, including RNA transcription, pre-mRNA processing, mRNA transport, localization, translation and stability [2
<ul><li>
found t  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17992</li><li>spans = 11562|11569</li></ul><li>
found t  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_17992</li><li>spans = 11562|11569</li></ul></ul>
<h2>CHEBI_23367</h2>
<h2>CHEBI_25213</h2>
<h2>CHEBI_25613</h2>
<h2>CHEBI_25676</h2>
<hr><p>Text source docID = 11604102<p>tting

Mouse tissues were homogenized in the RIPA lysis buffer (150 mM NaCl, 1.0% NP-40, 0.5% DOC, 0.1% SDS, 50 mM Tris, pH 8.0) at a concentration of<b> 0.2 g tissu</b>e per ml of buffer. The homogenized samples were cleared of debris by centrifugation at 10,000 × g for 10 minutes. Protein concentration of the tissue
<ul><li>
 0.2 g tissu  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_25676</li><li>spans = 18831|18843</li></ul><li>
 0.2 g tissu  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_25676</li><li>spans = 18831|18843</li></ul></ul>
<hr><p>Text source docID = 11604102<p> The insert of a Dazap1 cDNA clone P21, which encoded the C-terminal portion of DAZAP1 (starting from aa #197), was cloned in-frame into the EcoRI/Xho<b>I sites of a</b>n expression vector pET32b (Novagen, Madison, WI). Sequences at the junctions were verified by DNA sequencing. Milligrams of fusion proteins between t
<ul><li>
I sites of a  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_25676</li><li>spans = 17939|17951</li></ul><li>
I sites of a  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_25676</li><li>spans = 17939|17951</li></ul></ul>
<hr><p>Text source docID = 11604102<p>9 ammo acid residues of the mouse DAZAP1 was synthesized in vitro using the services of Bethyl Laboratories (Montgomery, TX). The peptide was conjugat<b>ed to KLH as</b> carrier and injected into a goat. The anti-DAZAP1-P antibody thus produced was purified on an affinity column containing the oligopeptide antigen.

W
<ul><li>
ed to KLH as  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_25676</li><li>spans = 18509|18521</li></ul><li>
ed to KLH as  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_25676</li><li>spans = 18509|18521</li></ul></ul>
<hr><p>Text source docID = 11604102<p>mino acid residue at the C-terminus. Both antibodies recognized in vitro synthesized DAZAP1 in an immunoprecipitation assay (data not shown). Western <b>blotting of </b>mouse tissue extracts detected a 45 kD protein that was present most abundantly in the testis, and to a lesser degree in spleen, liver, lung and brain
<ul><li>
blotting of   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_25676</li><li>spans = 9799|9811</li></ul><li>
blotting of   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_25676</li><li>spans = 9799|9811</li></ul></ul>
<h2>CHEBI_26020</h2>
<h2>CHEBI_26710</h2>
<hr><p>Text source docID = 11604102<p>DS, 50 mM Tris, pH 8.0) at a concentration of 0.2 g tissue per ml of buffer. The homogenized samples were cleared of debris by centrifugation at 10,00<b>0 × </b>g for 10 minutes. Protein concentration of the tissue extracts was determined by the Bradford method using the Bio-Rad Protein Assay system (Bio-Rad, 
<ul><li>
0 ×   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_26710</li><li>spans = 18936|18940</li></ul><li>
0 ×   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_26710</li><li>spans = 18936|18940</li></ul></ul>
<h2>CHEBI_27594</h2>
<h2>CHEBI_30879</h2>
<hr><p>Text source docID = 11532192<p>xcept that 5 days of training were used. The mice were analyzed in the same procedure room as was used for IOP measurment.

Clinical examinations

For<b> most s</b>trains, anterior chambers were examined with a slit lamp biomicroscope [16]. At least 8 mice of each strain shown in figure 1 were evaluated. However,
<ul><li>
 most s  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_30879</li><li>spans = 37929|37936</li></ul><li>
 most s  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_30879</li><li>spans = 37929|37936</li></ul></ul>
<hr><p>Text source docID = 11319941<p>les, A and B, or A and D. All stocks of mice were obtained from the Jackson Laboratory . ABF2 mice were generated at the University of Tennessee by Dr<b>. Riche</b>lle Strom [56] using Jackson Laboratory foundation stock. The F2 mice ranged in age from 35 to 143 days. The standard inbred strains ranged in age fro
<ul><li>
. Riche  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_30879</li><li>spans = 26319|26326</li></ul><li>
. Riche  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_30879</li><li>spans = 26319|26326</li></ul></ul>
<h2>CHEBI_32588</h2>
<hr><p>Text source docID = 11604102<p> in a Beckman SW41 rotor at 39,000 rpm for 2 hours at 4°C. Fractions of 0.5 ml were collected from the bottom of the tubes and analyzed by western blo<b>tti</b>ng.

Acknowledgments

We thank Gary Kuo for his involvement in the construction of Dazap1 expression vectors, and Ron Swerdloff's group for helpful di
<ul><li>
tti  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_32588</li><li>spans = 20325|20328</li></ul><li>
tti  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_32588</li><li>spans = 20325|20328</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ml of Rnasin ribonuclase inhibitor (Promega, Madison, WI), and a mixture of 10 protease inhibitors provided in the Protease Inhibitors Set (Roche Mole<b>cul</b>ar Biochemicals, Indianapolis, IN). Homogenates were centrifuged at 1,000 × g for 10 minutes to pellet cell debris and nuclei. After an additional cen
<ul><li>
cul  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_32588</li><li>spans = 19828|19831</li></ul><li>
cul  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_32588</li><li>spans = 19828|19831</li></ul></ul>
<h2>CHEBI_33521</h2>
<h2>CHEBI_33708</h2>
<h2>CHEBI_33709</h2>
<h2>CHEBI_35186</h2>
<hr><p>Text source docID = 11532192<p>n mice, the strain does not appear to be important as no early effect of anesthesia was present in the three distantly related laboratory strains [37]<b> we stud</b>ied in detail, and we have not observed any obvious effect during this period in any analyzed strain. Environmental differences may be important. Cage
<ul><li>
 we stud  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_35186</li><li>spans = 25549|25557</li></ul><li>
 we stud  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_35186</li><li>spans = 25549|25557</li></ul></ul>
<hr><p>Text source docID = 11532192<p>e strain does not appear to be important as no early effect of anesthesia was present in the three distantly related laboratory strains [37] we studie<b>d in de</b>tail, and we have not observed any obvious effect during this period in any analyzed strain. Environmental differences may be important. Cage cleanlin
<ul><li>
d in de  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_35186</li><li>spans = 25559|25566</li></ul><li>
d in de  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_35186</li><li>spans = 25559|25566</li></ul></ul>
<hr><p>Text source docID = 11532192<p> effect of anesthesia was present in the three distantly related laboratory strains [37] we studied in detail, and we have not observed any obvious ef<b>fect dur</b>ing this period in any analyzed strain. Environmental differences may be important. Cage cleanliness, changing frequency and housing density can alter
<ul><li>
fect dur  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_35186</li><li>spans = 25611|25619</li></ul><li>
fect dur  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_35186</li><li>spans = 25611|25619</li></ul></ul>
<h2>CHEBI_36080</h2>
<hr><p>Text source docID = 11604102<p>e segments of 35 bp, 133 bp and 90 bp that share 85%, 90%, and 97% similarity with segments in the human 3' UTR, respectively. These segments probably<b> contai</b>n regulatory elements.

The DAZAP1 protein contains two RNA-binding domains (RBDS) and a C-terminal portion that is rich in proline (Figure 1). It is 
<ul><li>
 contai  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4916|4923</li></ul><li>
 contai  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4916|4923</li></ul></ul>
<hr><p>Text source docID = 11604102<p>n vitro. It also bound to RNA homopolymers. We now report our characterization of the mouse Dazap1 gene and its protein product. The subcellular local<b>ization </b>of DAZAP1 suggests that it is not involved directly in mRNA translation.

Results

Characterization of the mouse Dazap1 cDNA

Mouse Dazap1 cDNA clones
<ul><li>
ization   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4184|4192</li></ul><li>
ization   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4184|4192</li></ul></ul>
<hr><p>Text source docID = 11597317<p>cient in their ability to repair the I-SceI-induced DSB through homologous recombination. Expression of I-SceI resulted in 1 out of 1400 cells produci<b>ng GFP </b>via homologous recombination in the human pancreatic tumor cell line CAPAN-1. CAPAN-1 cells carry a deletion of BRCA2 on one homolog and codes for a p
<ul><li>
ng GFP   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4454|4461</li></ul><li>
ng GFP   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4454|4461</li></ul></ul>
<hr><p>Text source docID = 11604102<p> levels in many tissues. Its predominant expression in testes suggests a role in spermatogenesis. In mouse testes, DAZAP1 was found both in the nuclei<b> and in</b> the cytoplasm. Its absence from polyribosomes indicates that it is not directly involved in mRNA translation.

Materials and methods

Isolation of mo
<ul><li>
 and in  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 15193|15200</li></ul><li>
 and in  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 15193|15200</li></ul></ul>
<hr><p>Text source docID = 11604102<p>nteraction with DAZ and DAZL, we isolated and characterized the mouse Dazap1 gene, and studied its expression and the subcellular localization of its <b>protein</b> product.

Results

The human and mouse genes have similar genomic structures and map to syntenic chromosomal regions. The mouse and human DAZAP1 prot
<ul><li>
protein  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 809|816</li></ul><li>
protein  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 809|816</li></ul></ul>
<hr><p>Text source docID = 11604102<p>y. The C-terminal proline-rich portions of DAZAP1 and Prrp are less conserved (81% similarity and 71% identity). There is an insertion of a 58 bp segm<b>ent in </b>Prrp cDNA that causes a change of reading frame and results in a shorter Prrp with a different C-terminal end sequence.

Figure 1

Evolutionary conser
<ul><li>
ent in   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5706|5713</li></ul><li>
ent in   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5706|5713</li></ul></ul>
<hr><p>Text source docID = 11604102<p>l amounts of total protein from various tissue extracts were applied to a 10% SDS-polyacrylamide gel and western blotted with the anti-DAZAP1-P antibo<b>dy.

Fi</b>gure 5

Western blot analyses of the expression of DAZAP1 and DAZL in mouse testes during postnatal development.

Subcellular localization of DAZAP1


<ul><li>
dy.

Fi  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10573|10580</li></ul><li>
dy.

Fi  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10573|10580</li></ul></ul>
<hr><p>Text source docID = 11604102<p>may facilitate the anchoring of Vg1 mRNA to the vegetal cortex. The Vg1 RNA encodes a member of the transforming growth factor-β family that is requir<b>ed for g</b>enerating dorsal mesoderm at the blastula stage of Xenopus embryogenesis [25]. Sequence conservation between the RBDs of DAZAP1 and Prrp suggests that
<ul><li>
ed for g  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 14459|14467</li></ul><li>
ed for g  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 14459|14467</li></ul></ul>
<hr><p>Text source docID = 11604102<p>cules, CA). About 50 μg of tissue extracts were separated on 10% SDS-polyacrylamide gels and blotted with either the anti-DAZAP1-C antibody (at a 1/2,<b>000 dil</b>ution) or the anti-DAZAP1-P antibody (at a 1/5,000 dilution). After incubation with horseradish peroxidase-conjugated secondary antibodies, the bindin
<ul><li>
000 dil  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 19243|19250</li></ul><li>
000 dil  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 19243|19250</li></ul></ul>
<hr><p>Text source docID = 11597317<p>t as a negative regulator of inappropriate RAD51–DNA interaction.

The final experiment reported by Davies et al [26] is particularly important with r<b>espect </b>to understanding the mechanism(s) through which BRCA2 promotes RAD51-dependent recombination. The localization of RAD51 was examined in CAPAN-1 cells,
<ul><li>
espect   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 8355|8362</li></ul><li>
espect   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 8355|8362</li></ul></ul>
<hr><p>Text source docID = 11604102<p> are associated with the polyribosomes [22].

The significance of the interaction between DAZAP1 and DAZL/DAZ remains to be defined. These proteins ma<b>y act t</b>ogether to facilitate the expression of a set of genes in germ cells. For example, DAZAP1 could be involved in the transport of the mRNAs of the targe
<ul><li>
y act t  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12395|12402</li></ul><li>
y act t  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12395|12402</li></ul></ul>
<hr><p>Text source docID = 11604102<p>line-rich portion, and the anti DAZAP1-P antibody was raised against an oligopeptide containing the last 19 amino acid residue at the C-terminus. Both<b> antibo</b>dies recognized in vitro synthesized DAZAP1 in an immunoprecipitation assay (data not shown). Western blotting of mouse tissue extracts detected a 45 
<ul><li>
 antibo  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 9690|9697</li></ul><li>
 antibo  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 9690|9697</li></ul></ul>
<hr><p>Text source docID = 11604102<p>the alternative splicing of the pre-mRNAs encoding neuronal inhibitory glycine receptor α2 (GlyR α2) [23]. The ability of Nova-1 to activate exon sele<b>ction i</b>n neurons is antagonized by a second RNA-binding protein, brPTB (brain-enriched polypyrimidine tract-binding protein), which interacts with Nova-1 and
<ul><li>
ction i  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12997|13004</li></ul><li>
ction i  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12997|13004</li></ul></ul>
<hr><p>Text source docID = 11604102<p>rdation protein [21]. Both FXR1P and FXR2P are associated with the polyribosomes [22].

The significance of the interaction between DAZAP1 and DAZL/DA<b>Z remai</b>ns to be defined. These proteins may act together to facilitate the expression of a set of genes in germ cells. For example, DAZAP1 could be involved 
<ul><li>
Z remai  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12353|12360</li></ul><li>
Z remai  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12353|12360</li></ul></ul>
<hr><p>Text source docID = 11604102<p>AZAP1 were prepared and purified on His-Bind metal chelation resins (Novagen, Madison, WI). The proteins were mixed with Freund's adjuvant and injecte<b>d into r</b>abbits to generate the anti-DAZAPl-C antibody. An oligopeptide containing the last 19 ammo acid residues of the mouse DAZAP1 was synthesized in vitro 
<ul><li>
d into r  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 18267|18275</li></ul><li>
d into r  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 18267|18275</li></ul></ul>
<hr><p>Text source docID = 11604102<p>inant expression in testes suggests a role in spermatogenesis. Its subcellular localization indicates that it is not directly involved in mRNA transla<b>tion.

</b>Background

Spermatogenesis is a complex developmental process in which male germ cells progress through mitotic proliferation, meiotic division and d
<ul><li>
tion.

  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1555|1562</li></ul><li>
tion.

  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1555|1562</li></ul></ul>
<hr><p>Text source docID = 11597317<p> highly conserved among mammalian orthologs [7,8].

Breast cancer susceptibility genes and DNA repair

Following the landmark discovery by Scully et a<b>l that </b>the homologous recombinase RAD51 colocalizes at subnuclear sites with BRCA1 [9], a number of additional results have provided evidence that both BRCA1
<ul><li>
l that   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1483|1490</li></ul><li>
l that   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1483|1490</li></ul></ul>
<hr><p>Text source docID = 11604102<p>adjuvant and injected into rabbits to generate the anti-DAZAPl-C antibody. An oligopeptide containing the last 19 ammo acid residues of the mouse DAZA<b>P1 was s</b>ynthesized in vitro using the services of Bethyl Laboratories (Montgomery, TX). The peptide was conjugated to KLH as carrier and injected into a goat.
<ul><li>
P1 was s  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 18397|18405</li></ul><li>
P1 was s  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 18397|18405</li></ul></ul>
<hr><p>Text source docID = 11604102<p>re identified through their interaction with another polysomal-associated RNA-binding protein, the fragile X mental retardation protein [21]. Both FXR<b>1P and F</b>XR2P are associated with the polyribosomes [22].

The significance of the interaction between DAZAP1 and DAZL/DAZ remains to be defined. These protein
<ul><li>
1P and F  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12233|12241</li></ul><li>
1P and F  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12233|12241</li></ul></ul>
<hr><p>Text source docID = 11604102<p>equences. However, a BLAST search of the GenBank for the 340 nt VLE sequence failed to identify any mammalian sequences with significant homology. Fur<b>ther map</b>ping of the RNA sequence within VLE that binds Prrp, and possibly DAZAP1, may help to identify the natural substrates of DAZAP1.

Conclusions

DAZAP1 
<ul><li>
ther map  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 14808|14816</li></ul><li>
ther map  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 14808|14816</li></ul></ul>
<hr><p>Text source docID = 11604102<p>itate the expression of a set of genes in germ cells. For example, DAZAP1 could be involved in the transport of the mRNAs of the target genes of DAZL.<b> Alterna</b>tively, DAZL and DAZAP1 may act antagonistically to regulate the timing and the level of expression. Such an antagonistic interaction between two inte
<ul><li>
 Alterna  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12568|12576</li></ul><li>
 Alterna  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12568|12576</li></ul></ul>
<hr><p>Text source docID = 11597317<p> which contains a restriction site for I-SceI, a yeast intron encoded endonuclease with an 18 base pair recognition site. Transient transfection of an<b> I-SceI</b> expression vector results in the production of a DSB in the first mutated copy of GFP. One or both DNA ends formed by the break invade(s) the homolog
<ul><li>
 I-SceI  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 3645|3652</li></ul><li>
 I-SceI  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 3645|3652</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ured. This line was found to have lower recombination efficiency than isogenic cells expressing full-length BRCA2 (the defect observed was about fivef<b>old to </b>sixfold). Second, in an independent study, Powell was able to compare derivatives of CAPAN-1 that differed only in their ability to express full-lengt
<ul><li>
old to   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5648|5655</li></ul><li>
old to   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5648|5655</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ino acids) offers very little clue to its function, although there are eight repeated segments (termed BRC repeats) in the middle of the protein that <b>are hig</b>hly conserved among mammalian orthologs [7,8].

Breast cancer susceptibility genes and DNA repair

Following the landmark discovery by Scully et al th
<ul><li>
are hig  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1330|1337</li></ul><li>
are hig  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1330|1337</li></ul></ul>
<hr><p>Text source docID = 11597317<p>d by CAPAN-1 cells was mislocalized to the cytoplasm, consistent with the fact that the truncation protein lacks a nuclear localization signal [28]. D<b>avies e</b>t al showed CAPAN-1 cells to be defective in nuclear localization of RAD51, raising the possibility that RAD51 is normally carried to the nucleus by b
<ul><li>
avies e  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 8808|8815</li></ul><li>
avies e  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 8808|8815</li></ul></ul>
<hr><p>Text source docID = 11597317<p> [16,17,18,19]. Biochemical analysis also showed interaction between BRCA1 and RAD51, although these detected interactions may have been indirect [9].<b> The BR</b>C repeats of BRCA2 are responsible for direct RAD51 interaction. Cells lacking BRCA1/2 fail to form damage-induced subnuclear RAD51 foci with normal e
<ul><li>
 The BR  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 2336|2343</li></ul><li>
 The BR  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 2336|2343</li></ul></ul>
<hr><p>Text source docID = 11604102<p>p1 gene and its protein product. The subcellular localization of DAZAP1 suggests that it is not involved directly in mRNA translation.

Results

Chara<b>cteriza</b>tion of the mouse Dazap1 cDNA

Mouse Dazap1 cDNA clones were isolated by library screening, and the 5' end of the cDNA was isolated by 5' RACE [15]. T
<ul><li>
cteriza  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4280|4287</li></ul><li>
cteriza  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4280|4287</li></ul></ul>
<hr><p>Text source docID = 11604102<p>. The expression of DAZAP1 during germ cell development paralleled that of DAZL (Figure 5). It is present at a low level in the testes of 6 days old m<b>ice whi</b>ch contained only primitive type A spermatogonia. The expression of DAZAP1 increased afterward, as the testes contained increasing number of prolifera
<ul><li>
ice whi  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10165|10172</li></ul><li>
ice whi  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10165|10172</li></ul></ul>
<hr><p>Text source docID = 11604102<p>tly in the testis, and to a lesser degree in spleen, liver, lung and brain (Figure 4). The protein was also present in the ovary. The expression of DA<b>ZAP1 du</b>ring germ cell development paralleled that of DAZL (Figure 5). It is present at a low level in the testes of 6 days old mice which contained only prim
<ul><li>
ZAP1 du  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10037|10044</li></ul><li>
ZAP1 du  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10037|10044</li></ul></ul>
<hr><p>Text source docID = 11597317<p>; ssDNA = single-strand DNA.

Figures and Tables

Figure 1

Recombination substrates used for assaying homology-directed repair. Cutting at the I-SceI<b> site w</b>ithin the mutant GFP (SceGFP) results in a double-strand break that can be repaired through homologous gene conversion using a 3'-truncated copy of GF
<ul><li>
 site w  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12981|12988</li></ul><li>
 site w  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12981|12988</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ely, between DAZAP1 and Prrp. These proteins may therefore have a similar RNA binding specificity. The C-terminal proline-rich portions of DAZAP1 and <b>Prrp are</b> less conserved (81% similarity and 71% identity). There is an insertion of a 58 bp segment in Prrp cDNA that causes a change of reading frame and res
<ul><li>
Prrp are  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5610|5618</li></ul><li>
Prrp are  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5610|5618</li></ul></ul>
<hr><p>Text source docID = 11597317<p>uired for the formation of recombinase complexes at the sites of DNA damage [20,21]. Finally, genetic measurements of recombination frequency have sho<b>wn that </b>Brca1-/- embryonic stem (ES) cells are deficient in recombinational DSB repair [22,23]. The similarity between phenotypes associated with BRCA1 and BR
<ul><li>
wn that   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 2692|2700</li></ul><li>
wn that   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 2692|2700</li></ul></ul>
<hr><p>Text source docID = 11597317<p>contacts [16,17,18,19]. Biochemical analysis also showed interaction between BRCA1 and RAD51, although these detected interactions may have been indir<b>ect [9]</b>. The BRC repeats of BRCA2 are responsible for direct RAD51 interaction. Cells lacking BRCA1/2 fail to form damage-induced subnuclear RAD51 foci with 
<ul><li>
ect [9]  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 2328|2335</li></ul><li>
ect [9]  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 2328|2335</li></ul></ul>
<hr><p>Text source docID = 11604102<p> were applied to a 10% SDS-polyacrylamide gel and western blotted with the anti-DAZAP1-P antibody.

Figure 5

Western blot analyses of the expression <b>of DAZA</b>P1 and DAZL in mouse testes during postnatal development.

Subcellular localization of DAZAP1

Our previous fractionation of mouse testis extracts sho
<ul><li>
of DAZA  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10628|10635</li></ul><li>
of DAZA  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10628|10635</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ypyrimidine tract-binding protein), which interacts with Nova-1 and inhibits its function [24]. DAZAP1 could function in a similar manner by binding t<b>o DAZL </b>and inhibiting its function. Comparing the phenotypes of Dazl1 and Dazap1 single and double knock-out mice may provide some clues to the significance 
<ul><li>
o DAZL   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 13237|13244</li></ul><li>
o DAZL   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 13237|13244</li></ul></ul>
<hr><p>Text source docID = 11597317<p>nuclear localization signal [28]. Davies et al showed CAPAN-1 cells to be defective in nuclear localization of RAD51, raising the possibility that RAD<b>51 is n</b>ormally carried to the nucleus by binding BRCA2. It is possible that RAD51 may alternatively be capable of BRCA2-independent transport, but mislocaliz
<ul><li>
51 is n  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 8923|8930</li></ul><li>
51 is n  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 8923|8930</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ipate in many cellular functions, including RNA transcription, pre-mRNA processing, mRNA transport, localization, translation and stability [20]. A ro<b>le for t</b>he DAZ family in the regulation of mRNA translation is supported by lines of circumstantial evidence, including the association of DAZL with polyribos
<ul><li>
le for t  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 11727|11735</li></ul><li>
le for t  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 11727|11735</li></ul></ul>
<hr><p>Text source docID = 11597317<p>e many cancer cells, genetically unstable and accumulate gross chromosomal rearrangements [5,6]. The sequence of this large protein (3418 amino acids)<b> offers</b> very little clue to its function, although there are eight repeated segments (termed BRC repeats) in the middle of the protein that are highly conser
<ul><li>
 offers  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1190|1197</li></ul><li>
 offers  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1190|1197</li></ul></ul>
<hr><p>Text source docID = 11604102<p>t is not involved directly in mRNA translation.

Results

Characterization of the mouse Dazap1 cDNA

Mouse Dazap1 cDNA clones were isolated by library<b> screen</b>ing, and the 5' end of the cDNA was isolated by 5' RACE [15]. The near fall length cDNA consists of a 53 bp 5' untranslated region (UTR), an open read
<ul><li>
 screen  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4367|4374</li></ul><li>
 screen  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4367|4374</li></ul></ul>
<hr><p>Text source docID = 11604102<p> different from two RNA-binding proteins, FXR1P and FXR2P, that were identified through their interaction with another polysomal-associated RNA-bindin<b>g prote</b>in, the fragile X mental retardation protein [21]. Both FXR1P and FXR2P are associated with the polyribosomes [22].

The significance of the interacti
<ul><li>
g prote  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12167|12174</li></ul><li>
g prote  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12167|12174</li></ul></ul>
<hr><p>Text source docID = 11319941<p>s known to affect brain development. One of these is Grk2, a member of the family of ionotropic glutamate receptor genes that is thought to play a rol<b>e in mo</b>dulating Huntington disease [44]. In the mouse, members of this receptor type act to indirectly down-regulate synaptic activity in the striatum [45]. 
<ul><li>
e in mo  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 21850|21857</li></ul><li>
e in mo  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 21850|21857</li></ul></ul>
<hr><p>Text source docID = 11597317<p> in suppressing unwanted RAD51–BRCA2 interactions.

Yet another potential role for BRCA2 in promoting RAD51-dependent recombinational repair is a posi<b>tive ro</b>le in assembly of RAD51 at damaged sites (Fig. 2b). Several proteins that interact with RAD51 are thought to 'mediate' assembly of RAD51 at sites of D
<ul><li>
tive ro  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10968|10975</li></ul><li>
tive ro  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 10968|10975</li></ul></ul>
<hr><p>Text source docID = 11604102<p> oligopeptide synthesized in vitro. The insert of a Dazap1 cDNA clone P21, which encoded the C-terminal portion of DAZAP1 (starting from aa #197), was<b> cloned</b> in-frame into the EcoRI/XhoI sites of an expression vector pET32b (Novagen, Madison, WI). Sequences at the junctions were verified by DNA sequencing.
<ul><li>
 cloned  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 17904|17911</li></ul><li>
 cloned  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 17904|17911</li></ul></ul>
<hr><p>Text source docID = 11604102<p> that causes a change of reading frame and results in a shorter Prrp with a different C-terminal end sequence.

Figure 1

Evolutionary conservation of<b> the DAZ</b>AP1 proteins. The amino acid sequences of the human and mouse DAZAP1s and the Xenopus Prrp are compared. The two RNA binding domains are boxed. Differ
<ul><li>
 the DAZ  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5872|5880</li></ul><li>
 the DAZ  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5872|5880</li></ul></ul>
<hr><p>Text source docID = 11319941<p> represents a cluster of linked genes, it will become appropriate to assess strengths of candidates using transgenic animals and by sequence compariso<b>ns [43]</b>.

Interval mapping places the QTL for Bsc10a in the central portion of Chr 10 in proximity with a number of genes known to affect brain development. 
<ul><li>
ns [43]  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 21580|21587</li></ul><li>
ns [43]  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 21580|21587</li></ul></ul>
<hr><p>Text source docID = 11604102<p>s are especially highly conserved. They share 98% and 97% similarity and 97% and 92% identity, respectively, between DAZAP1 and Prrp. These proteins m<b>ay there</b>fore have a similar RNA binding specificity. The C-terminal proline-rich portions of DAZAP1 and Prrp are less conserved (81% similarity and 71% identi
<ul><li>
ay there  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5506|5514</li></ul><li>
ay there  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5506|5514</li></ul></ul>
<hr><p>Text source docID = 11604102<p> and an autosome-encoded DAZ-like protein, DAZL. DAZAP1 contains two RNA-binding domains (RBDs) and a proline-rich C-terminal portion, and is expresse<b>d most </b>abundantly in the testis. To understand the biological function of DAZAP1 and the significance of its interaction with DAZ and DAZL, we isolated and c
<ul><li>
d most   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 549|556</li></ul><li>
d most   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 549|556</li></ul></ul>
<hr><p>Text source docID = 11604102<p>pus sequences are marked by #'s and *'s, respectively.

Genomic structure of Dazap1 and chromosomal mapping

Several overlapping lambda clones contain<b>ing mous</b>e Dazap1 genomic sequences were isolated. The locations of exons were determined by PCR amplification across exon-intron boundaries following by seque
<ul><li>
ing mous  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 6259|6267</li></ul><li>
ing mous  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 6259|6267</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ed later, in the cytoplasm of pachytene spermatocytes [5]. Genetic and biochemical studies suggest a role for the DAZ family in the regulation of mRNA<b> transla</b>tion. Drosophila Boule mutants was defective in the translation of the meiosis-specific CDC25 homologue, Twine [11], and DAZL was found to be associat
<ul><li>
 transla  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 3183|3191</li></ul><li>
 transla  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 3183|3191</li></ul></ul>
<hr><p>Text source docID = 11597317<p>s that interact with RAD51 are thought to 'mediate' assembly of RAD51 at sites of DNA damage (reviewed in [30]). RAD51 paralogs in particular, includi<b>ng XRCC2</b>, XRCC3, RAD51B, RAD51C and RAD51D, may function as a complex that actively promotes RAD51 assembly [31]. BRCA2 could cooperate with RAD51 paralogs in
<ul><li>
ng XRCC2  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 11192|11200</li></ul><li>
ng XRCC2  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 11192|11200</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ity factor, DAZ (Deleted in Azoospermia). In vitro, DAZAP1 interacts with both the Y chromosome-encoded DAZ and an autosome-encoded DAZ-like protein, <b>DAZL. D</b>AZAP1 contains two RNA-binding domains (RBDs) and a proline-rich C-terminal portion, and is expressed most abundantly in the testis. To understand the
<ul><li>
DAZL. D  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 442|449</li></ul><li>
DAZL. D  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 442|449</li></ul></ul>
<hr><p>Text source docID = 11597317<p>d at amino acid 1981 on the other homolog. The authors indicate that the level of I-SceI-induced recombination in CAPAN-1 is over 100-fold less than t<b>hat see</b>n using other (BRCA2+) human tumor cell lines. The different lines examined, however, are very likely to differ genetically from CAPAN-1 cells not onl
<ul><li>
hat see  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4776|4783</li></ul><li>
hat see  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 4776|4783</li></ul></ul>
<hr><p>Text source docID = 11604102<p>ron-specific nuclear RNA-binding protein, Nova-1. Nova-1 regulates the alternative splicing of the pre-mRNAs encoding neuronal inhibitory glycine rece<b>ptor α2 </b>(GlyR α2) [23]. The ability of Nova-1 to activate exon selection in neurons is antagonized by a second RNA-binding protein, brPTB (brain-enriched poly
<ul><li>
ptor α2   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12930|12938</li></ul><li>
ptor α2   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 12930|12938</li></ul></ul>
<hr><p>Text source docID = 11604102<p>e extracts was determined by the Bradford method using the Bio-Rad Protein Assay system (Bio-Rad, Hercules, CA). About 50 μg of tissue extracts were s<b>eparate</b>d on 10% SDS-polyacrylamide gels and blotted with either the anti-DAZAP1-C antibody (at a 1/2,000 dilution) or the anti-DAZAP1-P antibody (at a 1/5,00
<ul><li>
eparate  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 19142|19149</li></ul><li>
eparate  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 19142|19149</li></ul></ul>
<hr><p>Text source docID = 11604102<p>a-1 and inhibits its function [24]. DAZAP1 could function in a similar manner by binding to DAZL and inhibiting its function. Comparing the phenotypes<b> of Daz</b>l1 and Dazap1 single and double knock-out mice may provide some clues to the significance of their interaction. Dazl1 knock-out mice have already been
<ul><li>
 of Daz  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 13297|13304</li></ul><li>
 of Daz  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 13297|13304</li></ul></ul>
<hr><p>Text source docID = 11604102<p>tions and two deletions/insertions. The mammalian proteins shares 89% similarity and 81% identity with Xenopus Prrp (for proline-rich RNA binding prot<b>ein) [1</b>6]. The two RBDs are especially highly conserved. They share 98% and 97% similarity and 97% and 92% identity, respectively, between DAZAP1 and Prrp. T
<ul><li>
ein) [1  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5334|5341</li></ul><li>
ein) [1  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5334|5341</li></ul></ul>
<hr><p>Text source docID = 11604102<p>hat is most abundant in the testis. Although a majority of DAZAP1 is present in the cytoplasmic fraction, they are not associated with polyribosomes.
<b>
Conclus</b>ions

DAZAP1 is evolutionarily highly conserved. Its predominant expression in testes suggests a role in spermatogenesis. Its subcellular localization
<ul><li>

Conclus  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1338|1346</li></ul><li>

Conclus  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 1338|1346</li></ul></ul>
<hr><p>Text source docID = 11604102<p> DAZAP1 were generated. The anti DAZAP1-C antibody was raised against a recombinant protein containing the C-terminal proline-rich portion, and the an<b>ti DAZA</b>P1-P antibody was raised against an oligopeptide containing the last 19 amino acid residue at the C-terminus. Both antibodies recognized in vitro synt
<ul><li>
ti DAZA  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 9569|9576</li></ul><li>
ti DAZA  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 9569|9576</li></ul></ul>
<hr><p>Text source docID = 11597317<p>ll, personal communication, 2001). In these experiments, the derivative expressing full-length BRCA2 yielded 10-fold more recombinants than controls e<b>xpressi</b>ng only truncated BRCA2 (S. Powell, personal communication, 2001). While the experimental design of these experiments was somewhat different from that
<ul><li>
xpressi  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5979|5986</li></ul><li>
xpressi  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_36080</li><li>spans = 5979|5986</li></ul></ul>
<h2>CHEBI_37586</h2>
<hr><p>Text source docID = 11319941<p>ory . ABF2 mice were generated at the University of Tennessee by Dr. Richelle Strom [56] using Jackson Laboratory foundation stock. The F2 mice ranged<b> in age from 35 </b>to 143 days. The standard inbred strains ranged in age from 51 to 365 days. We studied approximately equal numbers of males and females.

Histological
<ul><li>
 in age from 35   <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_37586</li><li>spans = 26402|26418</li></ul><li>
 in age from 35   <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_37586</li><li>spans = 26402|26418</li></ul></ul>
<h2>CHEBI_6636</h2>
<hr><p>Text source docID = 11604102<p> SW41 rotor at 39,000 rpm for 2 hours at 4°C. Fractions of 0.5 ml were collected from the bottom of the tubes and analyzed by western blotting.

Ackno<b>wledg</b>ments

We thank Gary Kuo for his involvement in the construction of Dazap1 expression vectors, and Ron Swerdloff's group for helpful discussion. The w
<ul><li>
wledg  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_6636</li><li>spans = 20338|20343</li></ul><li>
wledg  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_6636</li><li>spans = 20338|20343</li></ul></ul>
<hr><p>Text source docID = 11604102<p>in ribonuclase inhibitor (Promega, Madison, WI), and a mixture of 10 protease inhibitors provided in the Protease Inhibitors Set (Roche Molecular Bioc<b>hemic</b>als, Indianapolis, IN). Homogenates were centrifuged at 1,000 × g for 10 minutes to pellet cell debris and nuclei. After an additional centrifugation 
<ul><li>
hemic  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_6636</li><li>spans = 19838|19843</li></ul><li>
hemic  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_6636</li><li>spans = 19838|19843</li></ul></ul>
<h2>CHEBI_8984</h2>
<hr><p>Text source docID = 11604102<p>d with either the anti-DAZAP1-C antibody (at a 1/2,000 dilution) or the anti-DAZAP1-P antibody (at a 1/5,000 dilution). After incubation with horserad<b>ish</b> peroxidase-conjugated secondary antibodies, the binding of antibodies was detected using the ECL Western Blotting System (Amersham Pharmacia Biotech,
<ul><li>
ish  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_8984</li><li>spans = 19342|19345</li></ul><li>
ish  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_8984</li><li>spans = 19342|19345</li></ul></ul>
<hr><p>Text source docID = 11604102<p>entration of 0.2 g tissue per ml of buffer. The homogenized samples were cleared of debris by centrifugation at 10,000 × g for 10 minutes. Protein con<b>cen</b>tration of the tissue extracts was determined by the Bradford method using the Bio-Rad Protein Assay system (Bio-Rad, Hercules, CA). About 50 μg of ti
<ul><li>
cen  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_8984</li><li>spans = 18969|18972</li></ul><li>
cen  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_8984</li><li>spans = 18969|18972</li></ul></ul>
<h2>CHEBI_9754</h2>
<hr><p>Text source docID = 11604102<p>l2, 0.3% NP-40, 40 U/ml of Rnasin ribonuclase inhibitor (Promega, Madison, WI), and a mixture of 10 protease inhibitors provided in the Protease Inhib<b>itor</b>s Set (Roche Molecular Biochemicals, Indianapolis, IN). Homogenates were centrifuged at 1,000 × g for 10 minutes to pellet cell debris and nuclei. Aft
<ul><li>
itor  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_9754</li><li>spans = 19807|19811</li></ul><li>
itor  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_9754</li><li>spans = 19807|19811</li></ul></ul>
<hr><p>Text source docID = 11604102<p>f 0.2 g tissue per ml of buffer. The homogenized samples were cleared of debris by centrifugation at 10,000 × g for 10 minutes. Protein concentration <b>of t</b>he tissue extracts was determined by the Bradford method using the Bio-Rad Protein Assay system (Bio-Rad, Hercules, CA). About 50 μg of tissue extract
<ul><li>
of t  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_9754</li><li>spans = 18980|18984</li></ul><li>
of t  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_9754</li><li>spans = 18980|18984</li></ul></ul>
<hr><p>Text source docID = 11604102<p>d centrifuged in a Beckman SW41 rotor at 39,000 rpm for 2 hours at 4°C. Fractions of 0.5 ml were collected from the bottom of the tubes and analyzed b<b>y we</b>stern blotting.

Acknowledgments

We thank Gary Kuo for his involvement in the construction of Dazap1 expression vectors, and Ron Swerdloff's group fo
<ul><li>
y we  <ul><li>Mike Bada</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_9754</li><li>spans = 20312|20316</li></ul><li>
y we  <ul><li>Kristin Garcia</li><li>class = http://www.owl-ontologies.com/unnamed.owl#CHEBI_9754</li><li>spans = 20312|20316</li></ul></ul>
